packages/event/src/dto/pub-sub.ts

```diff
@@ -0,0 +1,23 @@
+import { IsString, IsNotEmpty, IsEnum } from 'class-validator'
+import { PubSubTypeEnum, PubSubActionEnum } from '../enum/pub-sub'
+
+export class PubSubAttributes {
+  @IsString()
+  @IsNotEmpty()
+  gcp_events_project: string
```

Comment:
Nosso padrão normalmente é utilizar camelCase a nível de código.
Daí nesse caso ficaria:
```suggestion
  @Expose({ name: 'gcp_events_project' })
  gcpEventsProject: string
```

Reply:
Acoh que fiquei na duvida pq eu vi um created_at no codigo em algum lugar hahaha, mas tranquilo altero sim